README.md

```diff
@@ -1,70 +1,121 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# 📗 Table of Contents
 
-## Available Scripts
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-In the project directory, you can run:
+# 📖 Hello React Front End <a name="about-project"></a>
 
-### `npm start`
+**Hello React Front End** is a straightforward web application that merges the capabilities of fetching a random greeting from the backend API through this platform.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## 🛠 Built With <a name="built-with"></a>
+React, node, Visual Studio Code
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+### Tech Stack <a name="tech-stack"></a>
 
-### `npm test`
+- **React**
+- **Redux**
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### Key Features <a name="key-features"></a>
 
-### `npm run build`
+- **Random Greetings**
+- **API Integration**
+- **React Integration**
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## 💻 Getting Started <a name="getting-started"></a>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Setup <a name="setup"></a>
 
-### `npm run eject`
+To use this project, Clone this repository to your desired folder by pasting this command in your command line interface:
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+- `git clone https://github.com/binyamolango/hello-react-front-end.git`
+- `git clone https://github.com/binyamolango/hello-rails-back-end.git`
```

Comment:
- You have done well so far, though I can see the link to your backend here, but according to the project requirements you should have a link that leads directly to the backend repo, please fix this 👍 👍